### ISSUE TITLE
Does Pixel support embedding in other GUI widgets?

### ISSUE THREAD

**Issue by NaniteFactory** [faiface/pixel#130]
There are several GUI framewroks out there like Qt, GTK, etc. Pixel is a great library for 2D game developments and I want it to integrate with them. Is there a way to embed Pixel canvases into other windows of GUI libraries? Thanks.

**Comment by faiface** [comment]
It might be possible, but I haven't attempted that and don't really know how to do it.

**Comment by oref** [comment]
@NaniteFactory Did you ever get anywhere with this?

**Comment by NaniteFactory** [comment]
@oref No, I haven't got a clue. But it'd be great if someone had this feature implemented in Pixel..

**Comment by dusk125** [comment]
I've been working on embedding [imgui](https://github.com/ocornut/imgui) widgets into a pixel window (and using the pixel rendering stack). I'm ironing out the last few issues to get it ready for general use; [pixelui](https://github.com/dusk125/pixelui). If you were interested in embedding a different GUI framework, this could potentially be used as a guide; if you just wanted a gui system, this should just work for you.

**Comment by Asday** [comment]
Pixel renders with OpenGL and it seems like Qt supports embedding OpenGL views:  https://doc.qt.io/archives/qt-4.8/qt-opengl-hellogl-example.html

Does that help at all?
